app/config.sh

```diff
@@ -54,7 +54,7 @@ BUILD_PLATFORMS=""
 NUM_INCREMENTALS=6
 
 if [ "`uname`" = "Darwin" ]; then
-	alias mktemp='mktemp -t tmp'
+	alias mktemp='mktemp -t tmp.XXXXXXXXXX'
```

Comment:
Where exactly are you getting an error? This code only runs on macOS, and at least on current versions of macOS `-t` is for a prefix, so `-t tmp` is correct. It's also seemingly unnecessary, though — without `-t tmp` `-d` defaults to `-t tmp`, so this line could just be removed.

Reply:
It really can't work only with `-t tmp` on my macOS (Apple Silicon M1, MacBook Pro, Ventura 13.5.1)
<img width="1024" alt="image" src="https://github.com/zotero/zotero/assets/10293675/e21a7d1d-2f14-430b-8edf-928f971185af">
So this line could be removed.

Reply:
We can remove the line, but just to be clear, you've installed the GNU mktemp. This doesn't happen with the bundled BSD mktemp.

Reply:
I've removed the line in https://github.com/zotero/zotero/commit/e1def9971c18999b0e0bb127fa3f2cb1ccd59d5f. You can undo this change.

Reply:
Okey, I'm clear. Thanks:)